cmd/tfimport/main.go

```diff
@@ -120,14 +130,19 @@ func run() error {
 		log.Printf("Found importable resource: %q\n", ir.Change.Address)
 
 		// Attempt the import.
-		output, err := tfimport.Import(rn, ir, *inputDir, *terraformPath)
+		output, err := tfimport.Import(importRn, ir, *inputDir, *terraformPath)
+
+		// In dry-run mode, ignore the output, since nothing happened.
+		if *dryRun {
+			continue
```

Comment:
just curious, so in this case, we do `continue`, not `return`?

Reply:
This is inside the loop for every command, so we don't want to return prematurely. We expect the dry run to never fail, and its output and err are meaningless, so move on to the next resource instead of parsing them.

Reply:
Got it. Missed the loop part, was collapsed on the page.